apps/api/prisma/schema.prisma

```diff
@@ -319,5 +319,42 @@ model Document {
   fileSize             Int      @default(0)
   fileType             String?
 
+  documentMetadata DocumentMetadata? @relation(fields: [metadataId], references: [id])
+  metadataId       Int?              @unique
+
   @@unique([name, folderId])
 }
+
+model DocumentMetadata {
+  id                     Int       @id @unique @default(autoincrement())
+  version                Int
+  dateCreated            DateTime
```

Comment:
query for all devs really: do we want to be consistent on naming for things like date fields?  elsewhere we have used names like createdAt to hold the date created.  (Though tbh I think dateCreated is clearer!)

Reply:
While 'dateCreated' may be clearer, I agree that we should prioritize consistency, so using 'createdAt' makes sense for the time being.